Formula/arttime.rb

```diff
@@ -0,0 +1,28 @@
+class Arttime < Formula
+  desc "Clock, timer, time manager and ASCII+ text-art viewer for the terminal"
+  homepage "https://github.com/poetaman/arttime"
+  url "https://github.com/poetaman/arttime/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "385c8ddf39653ab52c1c1ea8edca14c19cb3eb05c8d1e6627201ccb2cc191755"
+  license "GPL-3.0-only"
+  head "https://github.com/poetaman/arttime.git", branch: "main"
+  
+  depends_on "fzf"
+  
+  on_linux do
+    depends_on "diffutils"
+    depends_on "less"
+    depends_on "libnotify"
+    depends_on "vorbis-tools"
+    depends_on "zsh"
+  end
+
+  def install
+    ENV["TERM"]="xterm"
+    system "./install.sh", "--noupdaterc", "--prefix", prefix, "--zcompdir", zsh_completion
+  end
+
+  test do
+    # arttime is a GUI application
+    system bin/"arttime", "--version"
```

Comment:
Can we at least check the output?

Reply:
Maybe when I push another version of arttime. At the moment the --version does not distinguish between release version and GitHub HEAD version. If I change the --version in between two releases, the test would fail for folks who install arttime with --HEAD.

Reply:
Those people are using an unsupported configuration so that does not matter

Reply:
@SMillerDev  Is --HEAD unsupported? Or you mean doing ``brew test arttime`` is unsupported for those who installed it with --HEAD?

Reply:
it does, if you do `brew install --HEAD arttime`, you can run `brew test arttime` and check it out.

Reply:
`--HEAD` installs are unsupported. If you do that you're on your own if anything stops working.